### PR TITLE
updated libraries to latest version / removed mlflow

### PR DESCRIPTION
**Pull request by mirkobronzi**

### CODE REVIEW

File: {{cookiecutter.project_slug}}/setup.py [hunk 1]
```diff
@@ -7,25 +7,24 @@
...
+        'pyyaml==6.0',
+        'pytest==7.1.2',
+        'pytest-cov==3.0.0',
         'jinja2<3.1.0',
         'sphinx==3.5.4',
         'sphinx-autoapi==1.8.1',
         'sphinx-rtd-theme==0.5.2',
         'sphinxcontrib-napoleon==0.7',
         'sphinxcontrib-katex==0.7.2',
         'recommonmark==0.7.1',
```
Comment (jerpint) [inline {{cookiecutter.project_slug}}/setup.py:25]: What about these versions? worth bumping?

Comment (mirkobronzi) [reply {{cookiecutter.project_slug}}/setup.py:25]: unfortunately doing so will results in `make html` not working anymore.
Googling around, it seems related to jinja2 (and this is the reason why it's set <3.1.0).

I am sure there is some way to improve all by finding the correct version combination; what I would do is to keep it as it is now (the html command works this way), and open a ticket for the future about fixing this. What do you think? (if you agree, you can open the ticket)

**Comment by jerpint** [comment]
No check has been run by the CI, is this intended? We should run a check before merging

**Review by jerpint — APPROVED** [review]
Looks good to me

**Pull request merged**